backend-models/service-tests/sectionedservice-test.txt

```diff
@@ -0,0 +1,551 @@
+sectionedservice.php
+
+----------------------------------
+getGroupsTest() line 100:
+----------------------------------
+
+*********
+ Pre-req
+*********
+
+(checklogin) IS TRUE
+
+//// PRE-REQ Values /////
+
+send
+{
+$opt = "GRP"
+$courseid = "1"
+$coursevers = "45656"
+}
+
+/////////////////////////
+
+
+*****************
+ Compare results
```

Comment:
No comparisons are to be made in this doc, only input and output are to be documented (and some times some pre-req data need to be created in the database)

Reply:
Ah so Compared results should be removed and send values will be put here?

Reply:
the comparisons should be removed from the document, as that is not the information that these test are created to retrieve. only the output is of interest. 

what i meant with input was the parameters that are sent to the service, which should be documented under the section "send"